src/main/java/com/synopsys/integration/jira/common/model/response/IssueTypeResponseModel.java

```diff
@@ -32,6 +35,7 @@
     private String name;
     private Boolean subtask;
     private Integer avatarId;
+    private Map<String, JsonElement> fields;
```

Comment:
Will we have some obvious (i.e. through this object or a service responsible for it) way of deserializing the values of this Map?

Reply:
I will be adding a simpler method of parsing these JsonElement types we have scattered through out the library. Saving and retrieving custom fields will be tough without it.